### PR TITLE
Kafka

### PR DESCRIPTION
**Pull request by alek-beloff**
Used avro and json parsers to parse schemas from schema registry. Only values are parsed, key schemas are not supported by collector

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,7 +9,9 @@ RUN apt-get update && \
     apt-get install -y -q build-essential \
     python3-dev  \
     libpq-dev \
-    curl
+    curl \
+    librdkafka-dev \
+    build-essential
```
Comment (Vixtir) [inline Dockerfile:14]: `build-essential` is already on 9th line

File: Dockerfile [hunk 2]
```diff
@@ -39,4 +41,4 @@ WORKDIR /app
 COPY . ./
 COPY --from=build /usr/local/lib/python3.9/site-packages /usr/local/lib/python3.9/site-packages
 
-ENTRYPOINT ["bash", "start.sh"]
\ No newline at end of file
+ENTRYPOINT ["/bin/bash", "-c", "sleep infinity"]
```
Comment (Vixtir) [inline Dockerfile:44]: why did you change an `entrypoint`?

File: collector_config.yaml [hunk 3]
```diff
@@ -1,11 +1,14 @@
+provider_oddrn: collector
 default_pulling_interval: 10
-token: ""
-platform_host_url: "http://localhost:8080"
+token: "AACLieplmVnlxN50mDXsHZ2Wy0oCFxjR7b5m67eZ"
```
Comment (Vixtir) [inline collector_config.yaml:3]: remove token

File: collector_config.yaml [hunk 4]
```diff
@@ -1,11 +1,14 @@
...
+platform_host_url: "http://localhost:8016"
```
Comment (Vixtir) [inline collector_config.yaml:4]: plz return default server port

File: collector_config.yaml [hunk 5]
```diff
@@ -1,11 +1,14 @@
...
 plugins:
-  - type: postgresql
-    name: test_psql
-    host: "database"
-    port: 5432
-    database: ""
-    user: ""
-    password: ""
+  - type: kafka
```
Comment (Vixtir) [inline collector_config.yaml:6]: return postgres example

File: odd_collector/adapters/kafka/adapter.py [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+logger = logging.getLogger('KafkaAdapter')
+logger.setLevel(logging.INFO)
+
+
+class Adapter(AbstractAdapter):
+
+    def __init__(self, config) -> None:
+        self.schema_registry_conf = config.schema_registry_conf
+        self.broker_conf = config.broker_conf
+        print(config.broker_conf)
```
Comment (Vixtir) [inline odd_collector/adapters/kafka/adapter.py:25]: Use logging.debug instead of print

**Review by Vixtir — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (alek-beloff) [reply Dockerfile:44]: Forgot to revert. That was temporarily, to start "start.sh" straight from docker container. Reverted it

File: Dockerfile (hunk 1 above)
Comment (alek-beloff) [reply Dockerfile:14]: good point. fixed
